wallet/src/Pos/Wallet/Web/Methods/Misc.hs

```diff
@@ -113,6 +117,22 @@ postponeUpdate = removeNextUpdate >> return NoContent
 applyUpdate :: (MonadWalletDB ctx m, MonadUpdates m) => m NoContent
 applyUpdate = removeNextUpdate >> applyLastUpdate >> return NoContent
 
+----------------------------------------------------------------------------
+-- System
+----------------------------------------------------------------------------
+
+-- | Triggers shutdown in a short interval after called. Delay is
+-- needed in order for http request to succeed.
+requestShutdown ::
+       ( MonadIO m
+       , MonadReader ctx m
+       , WithLogger m
+       , HasShutdownContext ctx
+       , Mockables m [Async, Delay]
+       )
+    => m NoContent
+requestShutdown = NoContent <$ async (delay (sec 1) >> triggerShutdown)
```

Comment:
This is dodgy.

Why does `triggerShutdown` cause the web server to fail to send a response?